components/api/db/cellery-hub-db.bal

```diff
@@ -184,3 +184,36 @@ public function searchOrganizations(string orgName, int offset, int resultLimit)
         return null;
     }
 }
+
+public function searchUserOrganizations(string userId, string apiUserId, string orgName, int offset, int resultLimit) 
+returns json | error {
+    log:printDebug(io:sprintf("Performing data retreival on REGISTRY_ORGANIZATION table for userId : %s, Org name : \'%s\': ",
+    userId, orgName));
+    table<record {}> resTotal = check connection->select(SEARCH_USER_ORGS_TOTAL_COUNT, gen:Count, orgName, userId);
+    json resTotalJson = check json.convert(resTotal);
+    int totalOrgs = check int.convert(resTotalJson[0]["count"]);
+    if (totalOrgs > 0){
+        log:printDebug(io:sprintf("%d organization(s) found with the orgName \'%s\' for userId %s", totalOrgs, orgName, userId));
+        map<any> imageCountMap = {};
+        table<gen:OrgListResponseImageCount> resImgCount = check connection->select(SEARCH_USER_ORGS_QUERY_IMAGE_COUNT, 
+        gen:OrgListResponseImageCount, userId, orgName, userId, orgName, apiUserId, orgName, resultLimit, offset);
+        foreach var fd in resImgCount{
+            imageCountMap[fd.orgName] = fd.imageCount;
+        }
+        table<gen:OrgListResponseAtom> res = check connection->select(SEARCH_USER_ORGS_QUERY, gen:OrgListResponseAtom, orgName, userId,
```

Comment:
Would this query work when there are more than one member in a org ? Could you please confirm ?

Reply:
Yes. It is working.